packages/react-ui-components/src/Headline/headline.spec.tsx

```diff
@@ -25,12 +26,6 @@ describe('<Headline/>', () => {
         expect(wrapper.prop('className')).toContain('fooClassName');
     });
 
-    it('should allow the propagation of additional props to the wrapper.', () => {
```

Comment:
`foo` is obviously no valid prop for `Headline`. Is this actually a wanted functionality?

Reply:
An html element may have so many possible attributes... I think this behavior was chosen in order to not have to enumerate all possible attributes.

Reply:
With TypeScript and React we can actually extend the interface (props) of a component with those of a HTML Element of our choice (eg. div).
See the Button component for an actual use case of this.